lib/handler.js

```diff
@@ -71,6 +72,8 @@ const handler = async (req, res, pgPool) => {
   } else if (req.method === 'GET' && segs.length === 0) {
     // health check - required by Grafana datasources
     res.end('OK')
```

Comment:
I like the idea of implementing the health check route as a shared platform route 👍🏻 

Let's remove the duplicate implementation from this file, please.

Reply:
Addressed in https://github.com/filecoin-station/spark-stats/pull/71/commits/368f30e381bb2434a1a48db942216675e36adaa5 (along with below comments)